pkgs/development/python-modules/torch/default.nix

```diff
@@ -1,5 +1,5 @@
 { stdenv, lib, fetchFromGitHub, fetchpatch, buildPythonPackage, python,
-  config, cudaSupport ? config.cudaSupport, cudaPackages,
+  config, cudaSupport ? config.cudaSupport, cudaPackages, autoAddDriverRunpath,
```

Comment:
Same as in the other PR: the current style is whatever it is but we don't have to make new arguments adhere to it (vote for one per line)